unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/Transform/Handler/ECSTransformHandler.cs

```diff
@@ -92,6 +92,11 @@ public void OnComponentModelUpdated(IParcelScene scene, IDCLEntity entity, ECSTr
 
             if (scaleChange || rotationChange)
                 sbcInternalComponent.OnTransformScaleRotationChanged(scene, entity);
+
+            // Same AvatarShape interpolation used at DCLTransform from SDK6
+            // We only want to trigger it if there's a method subscribed
+            if (entity.OnTransformChange != null)
+                entity.OnTransformChange.Invoke(model.position, model.rotation);
```

Comment:
just some sugar  - `entity.OnTransformChange?.Invoke(model.position, model.rotation);`

Reply:
Totally agreee, I didn't notice that, thanks!